roles/common/tasks/docker.yml

```diff
@@ -251,7 +251,7 @@
       lineinfile:
         dest: /etc/containerd/config.toml
         regexp: '^(.*)sandbox_image = "registry.k8s.io/pause(.*)$'
-        line: '\1sandbox_image = "{{ images_repo | default ("registry.k8s.io") }}/pause\2'
+        line: '\1sandbox_image = "{{ images_repo | default ("registry.k8s.io") }}/pause\2"'
```

Comment:
hi, does it work for you like that? for me it creates
```
sandbox_image = "registry.k8s.io/pause:3.6""
```
unless  i remove that quote.
the quote is part of the \2 backref. caught by (.*)$ on previous line. alternatively we can change the previous live to (.*)"$, does it make more sense to you?